tests/unit/nupic/algorithms/anomaly_likelihood_test.py

```diff
@@ -610,16 +654,185 @@ def testEstimationWindowSize(self):
     self.assertEqual(len(l._historicalScores), 3)
 
 
+  def testAnomalyLikelihoodWindowSizeImpactOnDistributionEstimateScoreArgs(
```

Comment:
@scottpurdy: it's actually more confusing than that: the `AnomalyLikelihoodTest` class contains test methods for the low-level functions as well as for the high-level `AnomalyLikelihood` class. Hence, I added the class name in my test methods to distinguish them from the the test methods that test the low-level functions. Perhaps it would be better to split the tests into two test case classes - something like `AnomalyLikelihoodClassTest` and AnomalyLikelihoodLowLevelTest`? Any suggestions?